src/Gravatar.php

```diff
@@ -38,11 +80,16 @@ public function __construct(array $defaults = [], bool $secure = true)
     /**
      * Returns an Avatar URL.
      */
-    public function avatar(string $email, array $options = [], bool $secure = null): string
+    public function avatar(string $email, array $options = [], bool $secure = null, bool $validateOptions = false): string
     {
         $url = 'avatar/'.$this->createEmailHash($email);
+
         $options = array_merge($this->defaults, array_filter($options));
 
+        if ($validateOptions) {
+            $options = $this->validateOptions($options);
```

Comment:
as I have said on the other comment if we throw exception then we do not need to return something here.

Reply:
Done and done.